.github/workflows/extension-ci.yml

```
@@ -0,0 +1,203 @@
+name: Extension CI
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)